libs/cart/driver/magento/src/cart.service.spec.ts

```diff
@@ -81,12 +80,9 @@ describe('Driver | Magento | Cart | CartService', () => {
 
     mockDaffCart = daffCartFactory.create();
     mockMagentoCart = magentoCartFactory.create();
-    mockMagentoCartItem = magentoCartItemFactory.create();
-    mockDaffCartItem = daffCartItemFactory.create();
+		mockDaffCartItem = daffCartItemFactory.create();
```

Comment:
This needed to change, because I added private fields to the `MagentoProductFactory` so the originalPrice, discount, and discountedPrices could be accurate with each other. The apollo testing module only returns the fields on the query, which do not include the private fields.